ompi/mca/common/ompio/configure.m4

```diff
@@ -0,0 +1,21 @@
+# -*- shell-script -*-
+#
+# Copyright (c) 2016      Research Organization for Information Science
+#                         and Technology (RIST). All rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+# MCA_ompi_common_ompio_CONFIG([action-if-can-compile],
+#                         [action-if-cant-compile])
+# ------------------------------------------------
+AC_DEFUN([MCA_ompi_common_ompio_CONFIG],[
+    AC_CONFIG_FILES([ompi/mca/common/ompio/Makefile])
+
+    AS_IF([test "$define_mpi_io" = "1"],
```

Comment:
Might be good to put a comment here explaining what `$define_mpi_io` is, and why it can enable/disable the entire common/ompi framework (I know you just copied the code from before, but that name is a bit... weird, and doesn't really follow the conventions of other enable/disable vars elsewhere in the code base).